speedcord/gateway.pyi

```diff
@@ -0,0 +1,15 @@
+from speedcord import Client
+from logging import Logger
+
+from .shard import DefaultShard
+
+
+class DefaultGatewayHandler:
+    client: Client
+    logger: Logger
+
+    def __init__(self, client: Client) -> None:
+        ...
+
+    async def on_receive(self, data: dict, shard: DefaultShard) -> None:
```

Comment:
gateway.py has this aswell which is bad. Maybe remove it from gateway.py?

Reply:
I'm not 100% what you're pointing at, is it the `dict` type annotation here:

```python
async def on_receive(self, data: -->dict<--, shard):
```

If so, then yeah I'll remove the duplicate type annotation from `gateway.py` - I tried to avoid making changes to the main `.py` files throughout all this.

Reply:
Yep exactly that! Feel free to update the .py files if needed!